lib/betagouvbot/notification_rule.rb

```diff
@@ -6,12 +6,16 @@ class NotificationRule
     attr_reader :horizon, :mail_file, :recipients
 
     class << self
+      def all
+        [in_three_weeks, in_two_weeks, in_one_day, one_day_after]
+      end
+
       def horizons
         all.map(&:horizon)
       end
 
-      def all
-        [in_three_weeks, in_two_weeks, in_one_day, one_day_after]
+      def find(horizon:)
```

Comment:
@Morendil That's called a "keyword argument". I chose to explicitly declare the attribute needed for the find, as it could be any other.

```ruby
# bof
NotificationRule.find(some_weird_variable); end

# yey
NotificationRule.find(horizon: some_weird_variable); end
```

Mostly for readability. Take a look: https://robots.thoughtbot.com/ruby-2-keyword-arguments